sdk/examples/client/node_api_core/20_get_milestone_by_index.rs

```diff
@@ -1,28 +1,38 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! Returns milestone data as JSON by its index by calling
-//! `GET /api/core/v2/milestones/by-index/{index}`.
+//! Returns milestone data as JSON by its index by querying the
+//! `/api/core/v2/milestones/by-index/{index}` node endpoint.
 //!
-//! `cargo run --example node_api_core_get_milestone_by_index --release -- [NODE URL]`
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --example node_api_core_get_milestone_by_index [MILESTONE INDEX] [NODE URL]
+//! ```
+
+use std::env;
 
 use iota_sdk::client::{Client, Result};
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
```

Comment:
No secrets here

Reply:
yeah, it's a bit annoying to be so explicit in this doc. I would like a more generic text that applies in any case.

Reply:
Changed.